canonical-web-and-design/vanilla-framework
# Flask titles

## Done

Added titles to all documentation pages and examples

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/
- Go to docs, make sure docs pages have `<title>` set
- Go to examples listing, make sure pages have `<title>` set
- Go to example themselves, make sure examples have `<title>` set
